src/test/java/com/google/sps/data/DrinkTest.java

```diff
@@ -0,0 +1,165 @@
+  }
+
+  @Test
+  public void testDrinkName() {
+    Assert.assertEquals(drink.getName(), DRINK_NAME);
+  }
+
+  @Test
+  public void testDrinkLong() {
+    Assert.assertEquals(drink.getID(), id);
+  }
+
+  @Test
+  public void testRoundToOneDecimalPlaceA() {
+    double num = 1.31;
+    double actual = drink.roundToOneDecimalPlace(num);
+    double expected = 1.3;
+    Assert.assertEquals(expected, actual, epsilon);
+  }
+
+  @Test
+  public void testRoundToOneDecimalPlaceB() {
+    double num = 1.999;
+    double actual = drink.roundToOneDecimalPlace(num);
+    double expected = 2.0;
+    Assert.assertEquals(expected, actual, epsilon);
+  }
+
+  @Test
+  public void testRoundToOneDecimalPlaceC() {
+    double num = 1.937;
+    double actual = drink.roundToOneDecimalPlace(num);
+    double expected = 1.9;
+    Assert.assertEquals(expected, actual, epsilon);
+  }
+
+  @Test
+  public void testRoundToOneDecimalPlaceD() {
+    double num = -1.937;
+    double actual = drink.roundToOneDecimalPlace(num);
+    double expected = 0.0;
+    Assert.assertEquals(expected, actual, epsilon);
+  }
+
+  @Test
+  public void testRoundToOneDecimalPlaceE() {
+    double num = 0.0;
+    double actual = drink.roundToOneDecimalPlace(num);
+    double expected = 0.0;
+    Assert.assertEquals(expected, actual, epsilon);
+  }
+
+  @Test
+  public void testUpdateAverageRatingA() {
+    double newRating = 1.0;
+    double actual = drink.updateAverageRating(ID_A, newRating);
+    double expected = 3.0;
+    Assert.assertEquals(expected, actual, epsilon);
+  }
+
+  @Test
+  public void testUpdateAverageRatingB() {
+    double newRating = 5.0;
+    double actual = drink.updateAverageRating(ID_B, newRating);
+    double expected = 2.6;
+    Assert.assertEquals(expected, actual, epsilon);
+  }
+
+  @Test
+  public void testUpdateAverageRatingC() {
+    double newRating = 2.0;
+    double actual = drink.updateAverageRating(ID_C, newRating);
+    double expected = 3.1;
+    Assert.assertEquals(expected, actual, epsilon);
+  }
+
+  @Test
+  public void testUpdateAverageRatingD() {
+    double newRating = 3.0;
+    double actual = drink.updateAverageRating(ID_D, newRating);
+    double expected = 1.9;
+    Assert.assertEquals(expected, actual, epsilon);
+  }
+
+  @Test
+  public void testUpdateAverageRatingE() {
+    double newRating = 4.0;
+    double actual = drink.updateAverageRating(ID_E, newRating);
+    double expected = 4.4;
+    Assert.assertEquals(expected, actual, epsilon);
+  }
+
+  @Test
+  public void testUpdateAverageRatingF() {
+    double newRating = 1.0;
+    double actual = drink.updateAverageRating(ID_F, newRating);
+    double expected = 1.0;
+    Assert.assertEquals(expected, actual, epsilon);
+  }
+
+  @Test
+  public void testUpdateAverageRatingG() {
+    double newRating = 0.0;
+    double actual = drink.updateAverageRating(ID_G, newRating);
+    double expected = ERROR_RETURN_VALUE;
+    Assert.assertEquals(expected, actual, epsilon);
+  }
+}
```

Comment:
A potential test case (also mentioned previously) is to test updating average rating for a new storeID.

Reply:
Ok I'll make a test case like that